src/utils/telephoneLink.jsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import A from '../atoms/a';
+
+const propTypes = {
+    className: PropTypes.string,
+    formattedNumber: PropTypes.string,
+    number: PropTypes.string,
+    tabIndex: PropTypes.number,
+};
+
+const defaultProps = {
+    className: undefined,
+    number: undefined,
+    formattedNumber: undefined,
+    tabIndex: -1,
+};
+
+function TelephoneLink(props) {
+    const {
+        className,
+        formattedNumber,
+        number,
+        tabIndex,
```

Comment:
Let's do an `id` prop for this guy as well.

Reply:
Fixed.